pages/linux/expr.md

```diff
@@ -0,0 +1,23 @@
+# expr
+
+> Evaluate expressions.
```

Comment:
From the examples this seems like a tool to manipulate strings, mostly, right? If so, the description should reflect that.